.github/workflows/chart-test.yaml

```diff
@@ -27,6 +27,16 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.6.1
 
+      - name: Symlink ci-lint -> ci
+        run: |
+          for dir in charts/*/; do
```

Comment:
Why not loop over `ci/*/lint` instead?

Reply:
Good question, I hadn't considered that now that we've got them all in one place.